apps/vr-tests/src/utilities/FabricDecorator.tsx

```diff
@@ -41,3 +41,25 @@ export const FabricDecoratorFullWidth: DecoratorFunction<StoryFnReactReturnType>
     </div>
   </div>
 );
+
+const mapper = { default: '', fixed: '300px', full: '100%' };
+export function modifyDeprecatedDecoratorStyles(options: {
```

Comment:
Is this temporary? If yes, can you please add a small TODO/notice?

Reply:
yes. I added explicit comments per story, but good idea to add it here as well. thx

Reply:
Done